### PR TITLE
added enbesa chains and icons

### PR DESCRIPTION
**Pull request by kruzuahz**
Added chains for enbasa shepards, elders and scholar, including images.
Added chains for Tourists, and consumed goods in old chains.
Added template for multifactories: orchards(old and new world),chemical plants(old/new), artisan workshops, assemblylines.
Added chains for high life dlc skyscraper consumables/ construction materials.
Added templates for venues: bar, cafe, restaurant, drugstore, furniture store, department store, iron tower.
Added a file for recipes and patents. I managed to fit them one page, but with only the final part of the chain. With the full chain you can fit about 8 on a single page, this gives a better overview I think.

I've added the chains as separate files to make it easier to read (feel free to copy them in if you wish)
Updated arctic icons to 124x124 pix images

Regards Kruzuahz

### CODE REVIEW

**Comment by dotSp0T** [comment]
Amazing, thanks for your work so far!! I will look into the PR as soon as I can find time (probably friday or the weekend). Should I just make the necessary adjustments or would you like a review and prompts on what to adjust further?

**Comment by kruzuahz** [comment]
You're welcome, no rush. Feel free to make the necessary adjustments. I tried to just add stuff and not change anything which was already there. a review of the numbers is also much appreciated.

**Comment by dotSp0T** [comment]
ToDos for ( @dotSp0T or @McDonnough ):

- [ ] check formatting
- [ ] fit into existing system
- [ ] review numbers

**Comment by dotSp0T** [comment]
ToDos for @kruzuahz :

- [ ] add themselves to collaborators list in the readme
- [ ] add themselves to collaborators section in the tex

File: main.tex
Comment (f1tzpatrick) [inline main.tex:1]: Do you want to update the version number too? (line 42)

File: main.tex [hunk 1]
```diff
@@ -998,31 +1122,88 @@ \section{Technician Consumables}
         \vspace{0.5\unitsize}}
 \end{tcbraster}
 % END Technician Consumables
-
 \tcbline
 
-\section{Electrification \& Symbols}
+\include{Tourist.tex}
```
Comment (f1tzpatrick) [inline main.tex:1127]: I tried to render this with `pdflatex`, the new pages appear blank. In the log, I saw a few import errors -

```
No file Tourist.tex.tex
...
No file enbesa.tex.tex.
No file shepards.tex.tex.
No file elders.tex.tex.
[5]
No file SkyGeneral.tex.tex.
No file SkyEngineers.tex.tex.
[6]
No file skyinvestors.tex.tex.
No file scholars.tex.tex.
[7]
No file recipes.tex.tex.
[8]
No file patents.tex.tex.
```

I was able to fix this by removing `.tex` from the new `\include{}` lines. For exmaple

`\include{Tourist.tex}` -> `\include{Tourist}`

After removing `.tex`, the new pages are populated with graphics :)

image

File: main.tex
Comment (f1tzpatrick) [reply main.tex:1]: Also, I'm not sure how to fix this, but I don't see the new sections in the Table of Contents 
image

**Comment by f1tzpatrick** [comment]
One last suggestion, can we make a `src/` folder to house the new `.tex` files? I really like how you've broken the sections out into their own files, but I think they would be more manageable in a subdirectory.

**Comment by kruzuahz** [comment]
Hello,

I'll try to answer all your comments:
- Version: I'll leave this to the maintainer of this repo to update if and when a release is made.
- Outliner: I use TEXStudio on windows, it works fine there
- Build problem: My compiler seems to be able to handle both with and without .tex. It uses this full command: pdflatex.exe -synctex=1 -interaction=nonstopmode "main".tex. 
- src folder: I'll leave this to the maintainer as well. Personally I don't think it's necessary given almost all the files are .tex. Also this will change the relative path to the icons which may give problems.

FYI i have added a lot more chains and info in my fork you might be interested in: https://github.com/kruzuahz/Anno_1800_Chains_Consumption/tree/dev. I decided not to PR this branch yet because it changes a lot and this PR has been stale for quite some time. 

Regards Kruzuahz

**Comment by f1tzpatrick** [comment]
Thanks @kruzuahz, seems like most of my issues stem from building with a different tool. Do you know if @dotSp0T used TEXStudio too? If so I can update my PR #10.

Thanks for sharing your dev branch too. It's a cool game and I think this document is nice to have nearby as a reference 🙂

**Comment by dotSp0T** [comment]
Hey you nice people! Thank you a thousand times for your work, and apologies i've not yet managed to approve the PRs. This has been a thing in the back of my head for quite some time now, but with work and everything it sorta lands on the end of the list every time I get a notification.

I'll do my best to look into both PRs on the weekend (fingers crossed) and have a look at the build-tool integration too.

Originally I/we did not include a build tool, etc. because it wasn't really needed. This started off as a project on Overleaf initially, then migrated into a purer LaTeX doc, using some VSCode and MIKTeX on windows as the build chain to generate the pdf version(s). Our goal was, and still is, to have a clean LaTeX source that can be compiled using any proper toolchain.

**Comment by dotSp0T** [comment]
@kruzuahz would you be cool, updating this PR with the rest of your work? If you like to we can even make you a/the owner of this repo - as you def have more time to work on this stuff rn

**Comment by kruzuahz** [comment]
@dotSp0T I've added a new PR which contains this work and the rest. I think I should abandon this PR and we can focus on that one. 

I'm willing to help (making a release, approving/merging the other PRs, updating issuelist etc) with this but the new PR has quite a bit of changes on the format (for better or worse). I'm a bit reluctant to just push it onto your project / work without your consent / feedback.

**Comment by dotSp0T** [comment]
@kruzuahz that sounds good. I'll be closing this one down, and go over the other one (do some adjustments if necessary) asap.

**Pull request closed without merging**